### ISSUE TITLE
Use TV Input List for adressable Light effects?

### ISSUE THREAD

**Issue by Jakohoho** [arachnetech/homebridge-mqttthing#391]
Is there a way to use the input list of a TV as a selection option for light effects of addressable LED strips? So to display the two devices as one in Homkit?

**Comment by NorthernMan54** [comment]
Something like this?

This is from homebridge-tasmota for a ws2812 capable led strip.



![22E63512-15B7-40B9-B373-9C85F719AE3B](https://user-images.githubusercontent.com/19808920/104367242-87a7d080-54e8-11eb-825d-731553e2fd48.png)

**Comment by Jakohoho** [comment]
yes, that's exactly what I've already done, but then you have two devices in Homekit.  I had seen this here: https://sprut.ai/client/article/2760 And was wondering if you could integrate that into the plugin, so that it can all be seen in one device?
